releasetool.yaml

```diff
@@ -0,0 +1,88 @@
+---
+projects:
+  vars:
+    oses:
+      - rhel7
+    signer: packages@theforeman.org
+    tar_projects: []
+    rpm_packages: []
+    koji_tags: []
+    artifact_strategy: 'rake'
```

Comment:
Artifact here means source archive? I think I could live with that. My concern is that its yet another term being used that won't necessarily be obvious to users coming from another area. For example, we use the the term `source` in our Jenkins jobs that generate sources. We use source terminology also in some of Obal's workflows. I don't so much care what we use, and more so that we end up with a consistent set of terminology given one of the goals is to land at a place where new comers can do these workflows.

Reply:
Yes, artifact means "whatever do we generate that presents the release to the users". So in most cases a tarball or a gem.

I think `source_strategy` works equally well. Shall I rename?

Reply:
Pending this change -- ACK. I generally like the direction of keep it all here for now but build it so it could be potentially be split in the future into a tool and configuration.